iModelCore/GeomLibs/geom/test/BsplineTest/t_bspBoundary.cpp

```diff
@@ -1380,1050 +1379,33 @@ TEST(bsptrim, MikeSurface)
-    points.insert(points.end(), {DPoint3d::From(0.20205994764569916,0.11855677775870209,0.017829892432499038), DPoint3d::From(0.2028037012952991,0.11816223410540261,0.017952229700200406), DPoint3d::From(0.20354745494489907,0.11776769045200127,0.018074566967899956), DPoint3d::From(0.20429120859449904,0.11737314679870178,0.018196904235599506), DPoint3d::From(0.20429864613099506,0.11736920136216879,0.0181981276082765), DPoint3d::From(0.20430608366749107,0.11736525592563579,0.018199350980953496), DPoint3d::From(0.20431352120398708,0.1173613104891028,0.01820057435363049), DPoint3d::From(0.2043383375590743,0.11734814600190607,0.01820465630344984), DPoint3d::From(0.20436315391416152,0.11733498151470935,0.018208738253269194), DPoint3d::From(0.20438797026924875,0.11732181702751263,0.018212820203088545)});
-    points.insert(points.end(), {DPoint3d::From(0.18496177766469919,0.11897703748480126,-0.0744018352523999), DPoint3d::From(0.18558821465599976,0.11858249383150178,-0.07491233284900045), DPoint3d::From(0.18621465164730034,0.1181879501782023,-0.07542283044570104), DPoint3d::From(0.1868410886386009,0.11779340652480096,-0.07593332804229977), DPoint3d::From(0.1868473530085139,0.11778946108826695,-0.07593843301826575), DPoint3d::From(0.1868536173784269,0.11778551565173294,-0.07594353799423173), DPoint3d::From(0.1868598817483399,0.11778157021519893,-0.07594864297019771), DPoint3d::From(0.18688078367308903,0.11776840572799882,-0.07596567641888753), DPoint3d::From(0.18690168559783816,0.11775524124079871,-0.07598270986757735), DPoint3d::From(0.18692258752258729,0.1177420767535986,-0.07599974331626718)});
-    points.insert(points.end(), {DPoint3d::From(0.09807983746729951,0.1193972972110023,-0.10976519872050085), DPoint3d::From(0.09811014639929999,0.11900275355760096,-0.11051833699930103), DPoint3d::From(0.09814045533119861,0.11860820990430148,-0.11127147527809939), DPoint3d::From(0.09817076426310087,0.11821366625100199,-0.11202461355689957), DPoint3d::From(0.09817106735241989,0.118209720814469,-0.11203214493968757), DPoint3d::From(0.0981713704417389,0.118205775377936,-0.11203967632247557), DPoint3d::From(0.09817167353105792,0.118201829941403,-0.11204720770526357), DPoint3d::From(0.0981726848299158,0.11818866545420628,-0.11207233719130576), DPoint3d::From(0.09817369612877368,0.11817550096700956,-0.11209746667734795), DPoint3d::From(0.09817470742763156,0.11816233647981284,-0.11212259616339014)});
-    points.insert(points.end(), {DPoint3d::From(0.034261370422198528,0.11973350499190201,-0.0848945914552005), DPoin
```

Comment:
wow! What are all these hard coded numbers?! I see they are used for testing. Why are the removed?

Reply:
:) This is an overkill test case that I moved to its own imjs file. I originally created this test before I knew how to read test inputs from a file, lol.